tests/unit/Actions.spec.ts

```diff
@@ -538,7 +536,7 @@ describe('Actions component - Filing Functionality', () => {
 
   it('Emits "Go To Dashboard" event when onClickCancel is called', async () => {
     const mockBuildFiling = jest.spyOn(wrapper.vm, 'buildIncorporationFiling')
-    const mockUpdateFiling = jest.spyOn(wrapper.vm, 'updateFiling')
+    mockUpdateFiling.mockReset().mockImplementation()
```

Comment:
I specific the default mock implementation again because some tests above set a specific implementation. (But doesn't really matter for the purposes of this test.)